internal/parser/parse_snippets_test.go

```diff
@@ -66,3 +67,20 @@ $ curl -X PATCH -H "Content-Type: application/json" localhost:8080/feedback/a02b
 	p5Snippet, _ := snippets5.Lookup("patch-feedback")
 	assert.Equal(t, []string{`curl -X PATCH -H "Content-Type: application/json" localhost:8080/feedback/a02b6b5f-46c4-40ff-8160-ff7d55b8ca6f/ -d '{"message": "Modified!"}'`}, p5Snippet.Lines())
 }
+
+func TestParameterSubstition_Snippets(t *testing.T) {
+	// Given
+	p6 := New([]byte(`
+Snippet without proper annotations:
+` + "```sh { name=run }" + `
+go run <FILENAME>
```

Comment:
We should test also multiple params, missing params, multiline scripts with params.